apps/pie-microsite/src/assets/styles/components/_list.scss

```diff
@@ -0,0 +1,50 @@
+@use '../fozzie-config' as f;
+
+.c-list {
+    list-style: none;
+    padding-left: 0;
+}
+
+.c-list--ordered {
```

Comment:
Bit of a nit, and might be worth seeing what @ashleynolan thinks. But I wonder if it makes more sense for the selector to be:

```
.c-list.c-list--ordered {
    // styles
}
```

That way the class modifier _has_ to be used in combination with the root/block class.

Totally subjective opinion!